src/locales/en.json

```diff
@@ -22,6 +22,19 @@
     "Empty": {
       "welcome": "Welcome on Cozy Notes",
       "after_created": "Once your Note is created, you can find it in your "
+    },
+    "SavingIndicator": {
+      "saved_just_now": "Saved just now.",
+      "saved_secs_ago": "Saved %{time} sec ago.",
+      "saved_min_ago": "Saved %{time} min ago.",
+      "saved_mins_ago": "Saved %{time} min ago.",
```

Comment:
```suggestion
      "saved_mins_ago": "Saved %{time} mins ago.",
```

Reply:
nope, AFAIK, we usually not put a "s" for "40 min" in english

Reply:
Interesting, [it seems a bit less strict than that](https://english.stackexchange.com/questions/125110/abbreviation-of-units-of-measure-hours-vs-minutes/125111) but `min` for `minutes` seems widely accepted!